src/adagrams.js

```diff
@@ -1,7 +1,96 @@
 const Adagrams = {
+  lettersPool: ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", "C", "D", "D", "D", "D", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "J", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N", "O", "O", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R", "R", "R", "R", "R", "S", "S", "S", "S", "T", "T", "T", "T", "T", "T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", "Y", "Z",],
+
   drawLetters() {
-    // Implement this method for wave 1
+    const hand = []
+    while (hand.length < 10) {
+      let tileNumber = Math.round(Math.random() * 98);
+      if (!hand.includes(tileNumber)) {
+        hand.push(tileNumber);
+      }
+    } 
+  return hand.map(x => this.lettersPool[x]);
+  },
+
+  usesAvailableLetters(input, drawnHand) {
+    const hash = {}
+
+    for (let i = 0; i < drawnHand.length; i++) {
+      hash[drawnHand[i]] != null ? hash[drawnHand[i]] += 1 : hash[drawnHand[i]] = 1
+    };
```

Comment:
This would be clearer if you used a `for (... of ...)` loop.  Also, you can simplify your ternary with a `||` instead:

```suggestion
    for (let letter of drawnHand) {
      hash[letter] = (hash[letter] || 0) + 1
    };
```